src/dmd/globals.d

```diff
@@ -114,6 +114,7 @@ struct Param
     bool isOpenBSD;         // generate code for OpenBSD
     bool isDragonFlyBSD;    // generate code for DragonFlyBSD
     bool isSolaris;         // generate code for Solaris
+    bool isLibcMusl;        // Use the Musl C Runtime
```

Comment:
We need a new place to put this information. Only the modules that take care of compilation steps care about this.

Reply:
I could have just defined a variable in the `dmd.mars` module, as it's only used there.  I simply added it to this global list, because it's where everything else is added.  Anyway, let's hold off on where to put the variable until we decide whether we want a runtime flag and what it's supposed to do first.

Reply:
Yes, hence just a comment from me. Generally speaking, all params that reference a platform or target should ideally not be here.